### PR TITLE
#181 - created reusable button component

### PR DESCRIPTION
**Pull request by kire21**
-  changed existing button component
- added setClass function in the utils folder

Screenshot from 2024-04-22 13-16-14

Screenshot from 2024-04-22 13-16-21

### CODE REVIEW

File: components/reusable-components/button/button.module.scss [hunk 1]
```diff
@@ -1,18 +1,56 @@
-.button {
-  padding: 10px 20px;
+.primaryButton {
+  display: flex;
+  gap: 8px;
+  background-color: var(--primary-btn-color);
+  color: var(--text-color);
+  border-radius: 8px;
+  padding: 8px 24px 8px 24px;
```
Comment (edichoska) [inline components/reusable-components/button/button.module.scss:7]: padding: 8px 24px

File: components/reusable-components/button/button.module.scss [hunk 2]
```diff
@@ -1,18 +1,56 @@
...
+.secondaryButton {
+  display: flex;
+  gap: 8px;
+  background-color: transparent;
   color: var(--text-color);
-  border: none;
-  border-radius: 5px;
+  border: 1px solid var(--primary-btn-color);
+  border-radius: 8px;
+  padding: 8px 24px 8px 24px;
```
Comment (edichoska) [inline components/reusable-components/button/button.module.scss:34]: same here, we can maybe have one class for the styles that both buttons share, and the primary button would have the .primaryButton for the styles that are unique to the primaryButton. same for secondary button

**Review by stoimenov95 — CHANGES_REQUESTED** [review]
Please resolve the comments. Everything else seems great.

**Comment by kire21** [comment]
Screenshot from 2024-04-23 11-49-37

Screenshot from 2024-04-23 11-49-11

Screenshot from 2024-04-23 11-49-05

Screenshot from 2024-04-23 11-48-53

File: app/page.module.scss [hunk 3]
```diff
@@ -3,226 +3,7 @@
   flex-direction: column;
   justify-content: space-between;
   align-items: center;
-  padding: 6rem;
+  // padding: 6rem;
```
Comment (stoimenov95) [inline app/page.module.scss:6]: Please remove unused comment.

**Review by stoimenov95 — APPROVED** [review]

**Review by stoimenov95 — APPROVED** [review]

**Review by edichoska — APPROVED** [review]
👏

**Pull request merged**